ci_scripts/build_suite.sh

```diff
@@ -24,6 +25,12 @@ if [ ! -e ${BUILD_DONE_FILE} ]; then
     echo "Building javascript code ..."
     yarn --ignore-engines --ignore-scripts --prod
     nodejs -e "try { require('fs').symlinkSync(require('path').resolve('node_modules/@bower_components'), 'g3w-admin/core/static/bower_components', 'junction') } catch (e) { console.log(e); }"
+
+    echo "Create unique django SECRET_KEY"
+    SECRET_KEY=$(python3 -c "from django.core.management.utils import get_random_secret_key;sk=get_random_secret_key();print (sk)")
+    touch ${BUILD_DONE_FILE}
+    echo ${SECRET_KEY} >> ${SECRET_KEY_FILE}
+
```

Comment:
Could we move this inside a dedicated management command? --> [Distributing Django projects with unique SECRET_KEYs](https://stackoverflow.com/questions/4664724/distributing-django-projects-with-unique-secret-keys)

Is it feasible to run these such of commands in a pre-installation phase?

Reply:
I can move the code inside a dedicated management command, as inside the stackoverflow  solution you which you mentioned, but I prefer to leave the file containing the secret key as a hidden file instead of a `.py` module.

What do you mean with 

> pre-installation phase

`build_suite.sh` is just in a _pre-installation_ phase, no?

Reply:
I mean, I don't know if we can run the `manage.py` commands before a django project is installed / created.

It's just a doubt, since I don't exactly know the whole flow..

Reply:
Ok, I just try it on a new docker deploy and it work fine.